frontend/src/components/editor/package-alert.tsx

```diff
@@ -214,6 +217,7 @@ const ProgressIcon = ({
 
 async function installPackages(
   packages: string[],
+  manager: "pip" | "uv" | "rye",
```

Comment:
Any TypeScript tips you can share to avoid retyping `"pip", "uv", and "rye"` here, in the zod schema, and elsewhere?

Reply:
I can fix this later, but fine to keep this for now